droidlet/lowlevel/robot_mover_utils.py

```diff
@@ -110,3 +111,32 @@ def xyz_pyrobot_to_canonical_coords(xyz):
 def xyz_canonical_coords_to_pyrobot_coords(xyz):
     """converts 3D coords from canonical to pyrobot coords."""
     return xyz @ np.linalg.inv(pyrobot_to_canonical_frame)
+
+def rotation_matrix_x(a):
```

Comment:
could we have a special file for rotations/spatial transforms outside of the generic mover_utils?  eventually would want to get to some unified spatial transform interface

Reply:
there is a file in `droidlet/lowlevel/rotation.py`. I'm moving these to that file.